OJP/OJP_Availability.xsd

```diff
@@ -13,16 +13,11 @@
 			<xs:documentation>Availability request. Used to check the availability of a MOBILITY SERVICE, a VEHICLE, SERVICE JOURNEY or SINGLE JOURNEY. Be aware that usually after the response arrives, the rest of the TRIP must be recalculated. As especially pick-up and drop-off locations may have shifted from where they were in the previously calculated TRIP.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:choice>
+			<xs:element name="PublicTransport" type="BookingPTLegStructure">
```

Comment:
This is a rather "interesting" case: Before the choice allowed to choose between a `<xs:element name="PublicTransport"` and a `<xs:annotation`.